src/lib/DepositFileUploader.js

```diff
@@ -0,0 +1,123 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2020 CERN.
+// Copyright (C) 2020 Northwestern University.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+// Drives the business logic of the InvenioFormApp.
+// Defines what happens when a button is clicked.
+
+import axios from 'axios';
+import _indexOf from 'lodash/indexOf';
+
+export class DepositFileUploader {
```

Comment:
silly question: do we still call all this `Deposit` or not it `Draft`?

Reply:
We kept the deposit as a concept and we use draft for the product that the deposit is managing. A deposit is creating/saving/publishing drafts!